### ISSUE TITLE
Search tool in column chooser

### ISSUE THREAD

**Issue by philrz** [brimsec/brim#975]
For addressing the problem described in #949, another approach proposed by the Dev team is to offer some kind of search tool. This could prove handy in situations where the number of fields is long enough that even searching through an alphabetized list might be tedious. This would also help with finding common substrings in the middle of field names. To offer an example with Zeek data, search for "tcp" would lead to field names like `active_tcp_conns`, `reassem_tcp_size`, and `tcp_conns`.

**Comment by r-franke** [comment]
Working on it.

**Comment by philrz** [comment]
Verified in Brim commit `7cf09ff`.

As shown in the attached video, this enables a use case where I've isolated just the Zeek `files` logs, and I want to pick a couple favorite fields, then easily find & include all the ones that have `uid` anywhere in the field names. Very handy!

[Verify.zip](https://github.com/brimsec/brim/files/5133339/Verify.zip)

That said, the verification made me think of a follow-on enhancement #1029 we may want to add at some point. But what we have here is already a huge improvement.

Thanks @r-franke!
